### PR TITLE
amendments to comply with Erlang/OTP version: 22

### PR DESCRIPTION
**Pull request by lucecisco**

### CODE REVIEW

File: src/erlang-lib/ec_genet/src/ec_genet_server.erl [hunk 1]
```diff
@@ -262,8 +261,7 @@ set_case(Tctx, HLPath, Choice, Case) ->
 %%% Mapping Function Interface
 %%%===================================================================
 
-throw_callmap_error(CallmapError,CallmapReason,Name) ->
-    Trace = erlang:get_stacktrace(),
+throw_callmap_error(CallmapError,CallmapReason,CallmapTrace,Name) ->
     FileLine =
         case Trace of
```
Comment (martin-volf) [inline src/erlang-lib/ec_genet/src/ec_genet_server.erl]: I think this cannot compile, the variable is called `CallmapTrace` in the function head.

**Review by martin-volf — CHANGES_REQUESTED** [review]

**Comment by martin-volf** [comment]
I think the change should not go to the master branch, since it would not compile/work with pre-5.4 NSO versions, it should go to a branch like nso-5.4 or something like that; @janlindblad, can you create the branch so that the pull-request can target it?

**Comment by janlindblad** [comment]
Created branch "pre-5.4" with latest commit as branch point. Martin, can you make the necessary changes?

**Comment by janlindblad** [comment]
Merged Martin's fix for OTP 22

**Pull request merged**